app/src/main/java/com/kickstarter/libs/utils/extensions/BackingExt.kt

```diff
@@ -14,6 +14,8 @@ fun Backing.isBacked(reward: Reward): Boolean {
 
 fun Backing.isErrored(): Boolean = this.status() == Backing.STATUS_ERRORED
 
+fun Backing.isErroredWithPLOT(): Boolean = this.status() == Backing.STATUS_ERRORED && !this.paymentIncrements().isNullOrEmpty()
```

Comment:
add as well here `this.status() == Backing.STATUS_AUTHENTICATION_REQUIRED` according to t[he last slack chat ](https://kickstarter.slack.com/archives/C07T8AXMQ9X/p1737573297144129?thread_ts=1737570138.719009&cid=C07T8AXMQ9X)